Remove git as a hard dependency and set rocblas_VERSION_COMMIT_ID
diff --git a/library/CMakeLists.txt b/library/CMakeLists.txt
index 9766be2..dbecfca 100644
--- a/library/CMakeLists.txt
+++ b/library/CMakeLists.txt
@@ -45,14 +45,7 @@ if( BUILD_VERBOSE )
 endif( )
 
 # Get the git hash of the hipBLAS branch
-find_package(Git REQUIRED)
-
-execute_process(COMMAND "${GIT_EXECUTABLE}" rev-parse HEAD
-                WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
-                OUTPUT_VARIABLE GIT_HASH_HIPBLAS
-                OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-set(hipblas_VERSION_COMMIT_ID "${GIT_HASH_HIPBLAS}")
+set(hipblas_VERSION_COMMIT_ID "unknown_guix")
 
 # log build commits
 message( STATUS "*** Building hipBLAS commit: ${hipblas_VERSION_COMMIT_ID}" )
